src/main/java/core/basesyntax/service/fileservice/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.fileservice;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    private static final String FILE_PATH = "src\\main\\resources\\report.csv";
+
+    @Override
+    public void writeToFile(String data) {
+        try (BufferedWriter fileWriter = new BufferedWriter(
+                new FileWriter(FILE_PATH, true))) {
```

Comment:
what does `true` stand for and do we need it?

Reply:
agree, it useless here
removed